tests/ReflectionTypeTest.php

```diff
@@ -0,0 +1,63 @@
+<?php
+namespace Go\ParserReflection;
+
+class ReflectionTypeTest extends \PHPUnit_Framework_TestCase
+{
+    protected function setUp()
+    {
+        if (PHP_VERSION_ID >= 70000) {
+            include_once (__DIR__ . '/Stub/FileWithClasses70.php');
+        }
+        if (PHP_VERSION_ID >= 70100) {
+            include_once (__DIR__ . '/Stub/FileWithClasses71.php');
+        }
+    }
+
+    /**
+     * Testing convertToDisplayType() with native \ReflectionType
+     *
+     * We're already testing it with Go\ParserReflection\ReflectionType
+     * elsewhere.
+     */
+    public function testTypeConvertToDisplayTypeWithNativeType()
+    {
+        if (PHP_VERSION_ID < 70000) {
+            $this->markTestIncomplete('PHP >= 7.0 implements ReflectionType');
+        }
+        $nativeClassRef = new \ReflectionClass('Go\\ParserReflection\\Stub\\ClassWithScalarTypeHints');
+        $nativeMethodRef = $nativeClassRef->getMethod('acceptsDefaultString');
+        $this->assertEquals(\ReflectionMethod::class, get_class($nativeMethodRef));
+        $nativeParamRefArr = $nativeMethodRef->getParameters();
+        $this->assertCount(2, $nativeParamRefArr);
+        $this->assertEquals(\ReflectionParameter::class, get_class($nativeParamRefArr[0]));
+        $nativeTypeRef = $nativeParamRefArr[0]->getType();
+        $this->assertEquals('string', (string)$nativeTypeRef);
+        $this->assertNotContains('\\', get_class($nativeTypeRef));
+        $this->assertInstanceOf(\ReflectionType::class, $nativeTypeRef);
+        $this->assertEquals('string', \Go\ParserReflection\ReflectionType::convertToDisplayType($nativeTypeRef));
+    }
+
+    /**
+     * Testing convertToDisplayType() with native \ReflectionType
+     *
+     * We're already testing it with Go\ParserReflection\ReflectionType
+     * elsewhere.
+     */
+    public function testTypeConvertToDisplayTypeWithNullableNativeType()
+    {
+        if (PHP_VERSION_ID < 70100) {
+            $this->markTestIncomplete('PHP >= 7.1 implements Nullable types');
```

Comment:
PHP 7.1+ implements **explicit** nullable types. I was considering including a 7.0+ test for implicit nullable types. It doesn't test a distinct code path, at least not in the code I was operating on here. Do you think I should add this test?

(By "implicit nullable types", I'm referring to parameters with class name type hints that default to `null`. I believe PHP 7.0+ is supposed to report those as `isNullable()`. I will double check this.)